Sources/AWSSDKSwiftCore/AWSClient.swift

```diff
@@ -192,143 +181,149 @@ extension AWSClient {
 // public facing apis
 extension AWSClient {
 
-    /// send a request with an input object and return a future with an empty response
+    /// execute a request with an input object and return a future with an empty response
     /// - parameters:
     ///     - operationName: Name of the AWS operation
     ///     - path: path to append to endpoint URL
     ///     - httpMethod: HTTP method to use ("GET", "PUT", "PUSH" etc)
+    ///     - serviceConfig: configuration to be used to request creation and signing
     ///     - input: Input object
     /// - returns:
     ///     Empty Future that completes when response is received
-    public func send<Input: AWSEncodableShape>(operation operationName: String, path: String, httpMethod: String, input: Input, on eventLoop: EventLoop? = nil) -> EventLoopFuture<Void> {
+    public func execute<Input: AWSEncodableShape>(operation operationName: String, path: String, httpMethod: String, serviceConfig: AWSServiceConfig, input: Input, on eventLoop: EventLoop? = nil) -> EventLoopFuture<Void> {
```

Comment:
Can we break this across multiple lines. It's a very long line eg
```
public func execute<Input: AWSEncodableShape>(
    operation operationName: String,
    path: String,
    ...
) -> EventLoopFuture<Void>
```

Reply:
sure we can. do you just want this specific part to be split to multiple lines or should I also adjust the other method (any hard-ish character limit per line?)

Reply:
We will be using swiftformat in the near future so these will be done automatically, but worthwhile sorting now